src/components/SearchMap.js

```diff
@@ -56,20 +65,94 @@ const SearchMap = ({mapLoaded, lastKnownRegion, updateRegion, sites, navigate})
+            const {id, title, description, nearestTown, accessibility, siteImageData, features, facilities} = selectedSite;
+            const {sitePreviewContainerStyle, touchableMainContainerStyle, mainInnerContainerStyle, topRowInfoStyle, topRowText, titleRowStyle, bottomRowInfoStyle, bottomRowText, closeIconStyle, IconContainer} = styles;
+
+            return (
+                <ImageBackground
+                    pointerEvents="none"
+                    source={siteImageData ? {uri: `data:image/png;base64,${siteImageData}`} : require('../../assets/starTent.jpg')}
+                    style={sitePreviewContainerStyle}
+                >
+                    <TouchableOpacity style={touchableMainContainerStyle} onPress={() => getSiteDetail({selectedSite: selectedSite, navigate})}>
+                        <View style={mainInnerContainerStyle}>
+                            <View style={topRowInfoStyle}>
+                                <View>
+
+                                </View>
+                                <Icon style={closeIconStyle} type='ionicon' name='md-close-circle' size={40}
+                                      color={'white'}
+                                      onPress={() => getSiteDetail({selectedSite: null})}/>
+                            </View>
+
+                            <View style={bottomRowInfoStyle}>
+                                <View style={titleRowStyle}>
+                                    <Text h4 style={bottomRowText}>{title}</Text>
+                                    <Text style={bottomRowText}> - </Text>
+                                    <Text style={bottomRowText}>{accessibility_options[accessibility]}</Text>
+                                </View>
+
+                                <View style={IconContainer}>
+                                    {renderIcons({features, facilities})}
+                                </View>
+                            </View>
+                        </View>
+                    </TouchableOpacity>
+
+                </ImageBackground>
+            );
+        }
+    };
+
     const renderMap = () => {
         if (mapLoaded) {
             return (
-                <MapView
-                    style={fillScreen}
-                    initialRegion={lastKnownRegion}
-                    onRegionChangeComplete={onRegionChange}
-                    rotateEnabled={false}
-                >
+                <View style={fillScreen}>
+                    <MapView
+                        style={fillScreen}
+                        initialRegion={lastKnownRegion}
+                        onRegionChangeComplete={onRegionChange}
+                        rotateEnabled={false}
+                    >
 
-                    {renderSites()}
+                        {renderSites()}
 
-                </MapView>
-            )
+                    </MapView>
+
+                    {renderSelectedSitePreview()}
```

Comment:
Note that `renderSelectedSitePreview` comes after the `MapView`. Below in the styles we use `absolute` positioning to have it sit over the top of the map.